evil-collection-buff-menu.el

```diff
@@ -27,10 +27,83 @@
 ;;; Bindings for `buff-menu'.
 
 ;;; Code:
+
+;; taken from emacs upstream repository with a minnor change
```

Comment:
Nit; In the future, capitalize / write with complete sentences.

Reply:
Note taken.